app/jobs/conversations/resolution_job.rb

```diff
@@ -0,0 +1,8 @@
+class Conversations::ResolutionJob < ApplicationJob
+  queue_as :medium
+
+  def perform(account:)
+    resolvable_conversations = account.conversations.open.select(&:auto_resolve?)
```

Comment:
This will load up all the open conversation objects and does computation in memory which might not be necessary in this case.  Instead, let's query from the database only the conversations that have passed the resolution period. 

feel free to clean up if required, move it to a scope etc.

```suggestion
    resolvable_time = Time.now.utc - account.auto_resolve_duration.days
    resolvable_conversations = account.conversations.open.where('last_activity_at < ? ', resolvable_time )
```